templates/repo/issue/view_title.tmpl

```diff
@@ -25,13 +25,18 @@
 	{{end}}
 
 	{{if .Issue.IsPull}}
+		{{ $baseTargetTag := printf "<a href=\"%s/src/branch/%s\">%s</a>" .RepoLink (EscapePound .BaseTarget) .BaseTarget }}
+		{{ $headTargetTag := printf "<code>%s</code>" .HeadTarget }}
+		{{if .Issue.PullRequest.HeadRepo}}
+			{{ $headTargetTag = printf "<a href=\"/%s/src/branch/%s\">%s</a>" .Issue.PullRequest.HeadRepo.FullName (EscapePound .Issue.PullRequest.HeadBranch) .HeadTarget }}
```

Comment:
This is not supported in golang 1.10. this logic should probably moved to go code (models/issue.go)

Reply:
Ah shoot. Didn't realize that was a 1.11 feature. Any reason you suggest to put it in issue.go rather than pull.go?

Reply:
Because base object is Issue and `BaseTarget` is property in `Issue` type if I'm not wrong

Reply:
But if `PullRequest` type has all needed data it can be added to pull.go

Reply:
I've been poking at this for a little while, and I'm super unsure about this - is putting markup in the Go the best idea? It certainly harms readability of the template, as there will be markup seemingly coming out nowhere. 

I tried to do something with a secondary template, but alas, [without passing a special function into our template](https://stackoverflow.com/questions/40164896/capture-or-assign-golang-template-output-to-variable), something like this isn't valid `{{ $headTargetTag := {{template "repo/issue/head_branch_link"}} }}`. I would like to avoid duplicating the logic between whether or not this is merged, and making a third translation item for this feels like overkill, at best. 

Certainly open to ideas but this is certainly a nasty result of the built in templating system.

Reply:
I was not thinking about moving markup in go but link and text generation parts. Markup is same in both cases

Reply:
It's not - one uses a `<code>` and one uses an `<a>`